### PR TITLE
Less verbose travis

### PR DESCRIPTION
**Pull request by vincentfretin**

### CODE REVIEW

**Comment by vincentfretin** [comment]
I'll merge if the tests pass.

File: .travis.yml [hunk 1]
```diff
@@ -33,16 +33,6 @@ install:
  - python2 $PWD/assembl/scripts/pypsql.py -u postgres -n localhost  "CREATE USER assembl_test WITH CREATEDB ENCRYPTED PASSWORD 'assembl_test'; COMMIT;"
  # The following command runs lots of sub-commands. After the following line has completed, approximately 10min have elapsed from the very beginning of the build
  - fab -c configs/travis.rc bootstrap_from_checkout
-before_script:
-  - source $VIRTUAL_ENV/bin/activate
-  # In jenkins, we were running the following commented steps at each build, in order to correctly update the already existing instance of Assembl (it was not built from zero each time). So should we do this for travis if we use more travis caching?
```
Comment (swergas) [inline .travis.yml]: No, don't remove this. It is a useful comment I wrote on purpose (please, before removing comments, ask yourself why their author spent time writing them). It is preliminary work we will refer to when we will try to optimize the travis build in a way that reuses a previously installed instance of Assembl.

Comment (vincentfretin) [reply .travis.yml]: I read your comments and giving the nature of travis where the environment is recreated each time and we already cache pip and node_modules, I thought those comments weren't necessary anymore. I don't think we can optimize further the tests time. I think database_delete app_update_dependencies app_compile_nodbupdate database_create bootstrap will take the same time as a fresh install.
I can put the comments back and comment the database_create and bootstrap commands that wasn't needed if you really want though.

Comment (vincentfretin) [reply .travis.yml]: https://github.com/assembl/assembl/commit/f370cbf5338e17db2cc7e3b9e396b7ae9b54e0ec

**Pull request merged**